design-proposals/cluster-orch-integrating-k8s-into-emt.md

```diff
@@ -0,0 +1,253 @@
+# Design Proposal: Integrating Kubernetes into Edge Microvisor Toolkit
```

Comment:
why are we melding together one-shot provsioning and embedding k3s in EMT? To me these are design proposals - also are we not going to support Ubuntu for the one-shot provisioning? Any thought on the impact of the UX?

Reply:
Agreed. The reason I put together was based on their complementary nature, as both aim to simplify and enhance the deployment process for Kubernetes at the edge.

About one-shot provisioning for Ubuntu, yes, we don't want a different user workflow for Ubuntu, although handling them internally can be different. Impact to UX is already under discussion.

Reply:
keep in mind that for ubuntu you dont have the manifest.json. This is only available for EMT images